### PR TITLE
ellipse documentation was corrected

### PR DESCRIPTION
**Pull request by warmspringwinds**
Documentation according to the issue https://github.com/scikit-image/scikit-image/issues/1476
was corrected.

### CODE REVIEW

File: skimage/measure/_regionprops.py [hunk 1]
```diff
@@ -364,8 +364,9 @@ def regionprops(label_image, intensity_image=None, cache=True):
         Coordinate list ``(row, col)`` of the region.
     **eccentricity** : float
         Eccentricity of the ellipse that has the same second-moments as the
-        region. The eccentricity is the ratio of the distance between its
-        minor and major axis length. The value is between 0 and 1.
+        region. The eccentricity is the ratio of the focal distance
+        (distance between focal points) over the major axis length.
+        The value is in the interval [0, 1). When it's 0, the ellipse becomes a circle.
```
Comment (stefanv) [inline skimage/measure/_regionprops.py]: Avoid contractions in docs: it's -> it is.

**Comment by stefanv** [comment]
@ahojnnes should check this--it is his implementation.

File: skimage/measure/_regionprops.py [hunk 2]
```diff
@@ -364,8 +364,9 @@ def regionprops(label_image, intensity_image=None, cache=True):
         Coordinate list ``(row, col)`` of the region.
     **eccentricity** : float
         Eccentricity of the ellipse that has the same second-moments as the
-        region. The eccentricity is the ratio of the distance between its
-        minor and major axis length. The value is between 0 and 1.
+        region. The eccentricity is the ratio of the focal distance
+        (distance between focal points) over the major axis length.
+        The value is in the interval [0, 1). When it is 0, the ellipse becomes a circle.
```
Comment (ahojnnes) [inline skimage/measure/_regionprops.py]: wrap at 80 please

**Comment by warmspringwinds** [comment]
@ahojnnes, done.

**Comment by JDWarner** [comment]
Thanks for the fix, @warmspringwinds!

**Pull request merged**